openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/explorePage.java

```diff
@@ -0,0 +1,179 @@
+package org.openmetadata.catalog.selenium.objectRepository;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class explorePage {
+  WebDriver webDriver;
+
+  public explorePage(WebDriver webDriver) {
+    this.webDriver = webDriver;
+  }
+
+  By explore = By.cssSelector("[data-testid='appbar-item'][id='explore']");
+  By tableCount = By.xpath("(//span[@data-testid='filter-count'])[1]");
+  By topicsCount = By.xpath("(//span[@data-testid='filter-count'])[2]");
+  By dashboardCount = By.xpath("(//span[@data-testid='filter-count'])[3]");
+  By pipelineCount = By.xpath("(//span[@data-testid='filter-count'])[4]");
+  By tables = By.xpath("(//button[@data-testid='tab'])[1]");
+  By topics = By.xpath("(//button[@data-testid='tab'])[2]");
+  By dashboard = By.xpath("(//button[@data-testid='tab'])[3]");
+  By pipeline = By.xpath("(//button[@data-testid='tab'])[4]");
+  By next = By.linkText("Next");
+  By pagination = By.xpath("//div[@data-testid=\"pagination-button\"]");
+  By bigQueryCheckbox = By.xpath("//input[@id=\"BigQuery\"]");
+  By serviceName = By.xpath("//div[@class=\"filters-title tw-w-40 tw-truncate custom-checkbox-label\"]");
```

Comment:
Can you replace `xpath` with `data-testid`. If data-testid is not present, can you please add it.

Comment applies to other instances of this in this PR